src/configuration/appDefaults.js

```diff
@@ -0,0 +1,16 @@
+const DASHBOARD_CONFIG = {
+  landingPath: '/win10/overview?numDays=90',
+  timeRange: 90, // 90 days
+  colors: ['#e55525', '#ffcd02'],
+  labels: ['Firefox', 'Chrome'],
+};
+
+export const SETTINGS = {
+  frameworks: {
```

Comment:
Move `frameworks` into DASHBOARD_CONFIG;
Define in DASHBOARD_CONFIG a 'platform' key like this:
```javascript
 platforms: {
  linux64: {
   label: 'Linux64',
   benchmarks: Linux64Config.benchmarks,
  },
  etc,
}
```

This is very similar to what you suggested and you show that you have a grasp of what is involved.